examples/07_cloudflare/waku.cloudflare-middleware.ts

```diff
@@ -0,0 +1,74 @@
+import type { Middleware } from 'waku/config';
+import type { Env } from 'hono';
+import { getHonoContext as _getHonoContext } from 'waku/unstable_hono';
+
+export type HandlerReq = {
+  body: ReadableStream;
+  url: URL;
+  method: string;
+  headers: Record<string, string>;
+};
+export type HandlerRes = {
+  body?: ReadableStream;
+  headers?: Record<string, string | string[]>;
+  status?: number;
+};
+export type HandlerContext = {
+  readonly req: HandlerReq;
+  readonly res: HandlerRes;
+  readonly context: Record<string, unknown>;
+};
```

Comment:
Can we export this from waku? I think only the Middleware type is exported from `'waku/config'` and I don't see a way to extract HandlerContext.

Reply:
I'll consider it in the future, but not soon.

Meanwhile:
```ts
type HandlerContext = Parameters<ReturnType<Middleware>>[0];
```

Reply:
Thanks. I don't think I've used `<Parameters>` before. Good to know.